.travis.yml

```diff
@@ -7,6 +7,8 @@ services:
 cache: bundler
 before_install:
   - export TZ=Europe/London
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
```

Comment:
This looks like a hack, I think we should use the bundler version used to build the Gemfile.